moved php to docs-src, added info to legacy pages (#1602)
temporalio/documentation: docs/php/activities.md

sidebar_label: Activities
---

:::info Try the Application development guide

The majority of this information has moved into the [Application development guide](/application-development/?lang=php).

> However, if you can not find what you are looking for there then we recommend checking this doc set as well.

Reviewer: ```suggestion
However, if you can't find what you are looking for there, we recommend checking this doc set as well.
```
